src/test/files/checks/htl/CamelCaseCheck.html

```diff
@@ -0,0 +1,13 @@
+<sly data-sly-template.COMPONENT_VALIDATION></sly> <!--/* Non-Compliant */-->
+<sly data-sly-use.GalleryModel="com.example.GalleryModel"></sly> <!--/* Non-Compliant */-->
+<sly data-sly-template.componentValidation></sly>
+<sly data-sly-use.galleryModel="com.example.GalleryModel"></sly>
+<sly data-sly-use.model="com.example.GalleryModel"></sly>
+<sly data-sly-use.GalleryModel="com.example.GalleryModel"></sly> <!--/* Non-Compliant */-->
+<sly data-sly-use.gallery_model="com.example.GalleryModel"></sly> <!--/* Non-Compliant */-->
+<sly data-sly-use.gallery-model="com.example.GalleryModel"></sly> <!--/* Non-Compliant */-->
+<sly data-sly-use.gallery_Model="com.example.GalleryModel"></sly> <!--/* Non-Compliant */-->
+<sly data-sly-use.gallery-Model="com.example.GalleryModel"></sly> <!--/* Non-Compliant */-->
+<sly data-sly-use.DAMAsset="com.example.DamAsset"></sly> <!--/* Non-Compliant */-->
+<sly data-sly-use.damAsset="com.example.DamAsset"></sly>
+<sly data-sly-use.assetFromDAM="com.example.DamAsset"></sly> <!--/* Non-Compliant */-->
```

Comment:
Is this test works with current pattern?

Reply:
I've fixed regexp.